lib.py

```diff
@@ -51,12 +67,41 @@ def get_inventory():
     if type(output) is bytes:
         output = output.decode()
     if not p.returncode:
-        return json.loads(output)
+        inventory = json.loads(output)
+        if cache_file:
+            write_inventory_to_cache(cache_file=cache_file, inventory=inventory)
+        return inventory
     else:
         logging.error(error)
         raise Exception("failed to query ansible-inventory")
 
 
+def get_inventory_from_cache(cache_file, cache_timeout):
+    """Read ansible-inventory from cache file
+
+    :return cache
```

Comment:
I reckon the docstring in sphinx format should be `:return: cache`.
```suggestion
    :return: cache
```

Reply:
I have updated the docstring to be compliant, thanks!